xExtension-YouTube/extension.php

```diff
@@ -180,11 +200,14 @@ public function getHtmlPeerTubeContentForLink(FreshRSS_Entry $entry, string $lin
 	public function getHtml(FreshRSS_Entry $entry, string $url): string
 	{
 		$content = '';
+	
+		if ($this->autoSize) {
+			$iframe_attribute = 'class="yt_player_autosize"';
+		} else {
+			$iframe_attribute = 'width="' . $this->width . '" height="' . $this->height . '"';
```

Comment:
I can't remember why the style attribute was used, but I am almost certain, that I added it on purpose. 
It's gone now.